### PR TITLE
switching to results tab

### PR DESCRIPTION
**Pull request by pflopez**
PLAN-364

- Switches to results tab when the user clicks generate
- Disables form controls after user click generate

### CODE REVIEW

File: src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts [hunk 1]
```diff
@@ -406,6 +392,12 @@ export class CreateScenariosComponent implements OnInit, OnDestroy {
     // });
   }
 
+  disableForms() {
+    this.formGroups.forEach((form) => {
+      form.disable();
```
Comment (sachiyuan) [inline src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts:397]: When the forms are disabled but the inputs are still there (aka there is a value input for name, max slope, etc) is the Generate button still enabled? If not, can we also disable that button here so the user can't go back and spam create_scenario requests?

Comment (pflopez) [reply src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts:397]: good call! Will disable

Comment (pflopez) [reply src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts:397]: Ah actually is already disabled :D 

```
  createScenario(): void {
    this.generatingScenario = true;
    // TODO Add error catching for failed scenario creation
    this.planService
      .createScenario(this.formValueToScenario())
      .subscribe((_) => {
        const planId = this.plan$.getValue()?.id;
        this.scenarioState = 'pending';
        this.disableForms();
        this.selectedTabIndex = 1;
      });
```

```
<button
                mat-raised-button
                color="primary"
                [disabled]="
                  !formGroups[0].valid ||
                  !formGroups[1].valid ||
                  !formGroups[2].valid ||
                  !formGroups[3].valid ||
                  generatingScenario
                "
                (click)="createScenario()">
                {{ generatingScenario ? 'GENERATING SCENARIO...' : 'GENERATE' }}
              </button>
```

Comment (sachiyuan) [reply src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts:397]: Oh! Right, forgot that condition also used the generatingScenario boolean. Go ahead and ignore me then

**Review by sachiyuan — APPROVED** [review]
Lgtm!

**Pull request merged**